package.json

```diff
@@ -3,7 +3,9 @@
   "version": "0.0.1",
   "description": "Testing 123",
   "main": "index.js",
-  "files": ["index.js"],
+  "files": [
+    "index.js"
```

Comment:
Could we point this to the new tokens folders?

Reply:
donezo